packages/collector/test/tracing/database/db2/app.js

```diff
@@ -34,20 +36,32 @@ const connStr2 =
  *
  * We are currently using the IBM DB2 cloud service, because we had trouble on Circleci.
  *
- * Example connection string for ENV:
- * DATABASE=bludb;HOSTNAME=*.databases.appdomain.cloud;UID=msv01866;PWD=xxx;PORT=31198;PROTOCOL=TCPIP;SECURITY=SSL
+ * The db2 instance is hosted on IBM Cloud. To test the cloud instance locally, you need to enable
+ * public endpoint temporarly:
+ *
+ * - Go to https://cloud.ibm.com/resources. Click on your instance.
```

Comment:
It seems I don't have sufficient permissions to check the "Connections" section on the UI. Is there an alternative way we can test this? Also, can we store this in 1Password? I came across `Team Node.js: DB2 IBM Cloud Instance`, but it doesn't appear to be working in local.

Reply:
We cannot put the public creds into 1pwd, because the public endpoint is removed. These instructions will help the team in case we have to test it locally against the remote db2 instance.
Locally, we test against docker by default.

> It seems I don't have sufficient permissions

Please ask Chris 👍

Reply:
Verified the tests 👍